core/settings.py

```
@@ -0,0 +1,732 @@
+                    },
+                },
+                "TIMEOUT": 300,
+            },
+        }
+    )
+    SELECT2_CACHE_BACKEND = "select2"
+
+    EMAIL_HOST = values.Value(
+        environ=True, environ_prefix=None, environ_name="EMAIL_SERVER"
+    )
+    EMAIL_USE_TLS = values.BooleanValue(True, environ=False)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)